base/mapiterator.jl

```diff
@@ -0,0 +1,175 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+
+"""
+    MapIterator(f, c...) -> iterator
+
+`collect(MapIterator(f, c...))` is equivalent to `map(f, c...)`
+"""
+
```

Comment:
Is this a hard requirement to make the docstrings machine-readable? or is it an issue of style?